src/arch/riscv/machine/hardware.c

```diff
@@ -273,5 +273,5 @@ BOOT_CODE void initIRQController(void)
 static inline void handleSpuriousIRQ(void)
 {
     /* Do nothing */
-    printf("Superior IRQ!! SIP %lx\n", read_sip());
+    printf("Superior IRQ!! SIP 0x%lx\n", read_sip());
```

Comment:
Could take the chance to also use the proper format:
```suggestion
    printf("Superior IRQ!! SIP 0x%" SEL4_PRIx_word "\n", read_sip());
```